server/src/routes/upload/parseBeatmap.ts

```diff
@@ -114,11 +134,11 @@ export const parseBeatmap: (
       bpm: infoJSON._beatsPerMinute,
 
       difficulties: {
-        easy: difficulties.some(x => x._difficultyRank === 1),
-        expert: difficulties.some(x => x._difficultyRank === 7),
-        expertPlus: difficulties.some(x => x._difficultyRank === 9),
-        hard: difficulties.some(x => x._difficultyRank === 5),
-        normal: difficulties.some(x => x._difficultyRank === 3),
+        easy: difficultiesData[0],
```

Comment:
This isn't the most readable. Consider refactoring the variables on Line 105 to use destructuring.